subsys/bluetooth/mesh/beacon.c

```diff
@@ -104,11 +138,13 @@ static void secure_beacon_create(struct bt_mesh_subnet *sub,
 	/* IV Index */
 	net_buf_simple_add_be32(buf, bt_mesh.iv_index);
 
-	net_buf_simple_add_mem(buf, sub->auth, 8);
+	net_buf_simple_add_mem(buf, sub->secure_beacon.auth, 8);
 
 	LOG_DBG("net_idx 0x%04x flags 0x%02x NetID %s", sub->net_idx, flags,
 		bt_hex(keys->net_id, 8));
-	LOG_DBG("IV Index 0x%08x Auth %s", bt_mesh.iv_index, bt_hex(sub->auth, 8));
+	LOG_DBG("IV Index 0x%08x Auth %s", bt_mesh.iv_index, bt_hex(sub->secure_beacon.auth, 8));
+
+	return 0;
```

Comment:
is it really required?

Reply:
Yes, to align signature of `secure_beacon_create` and `private_beacon_create`